### PR TITLE
Spur context api

### PR DESCRIPTION
**Pull request by jjunqueira**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/SpurContextAPI/Integrations/SpurContextAPI/SpurContextAPI.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+            result = test_module(client)
+            return_results(result)
+
+        elif demisto.command() == 'spur-context-api-enrich':
+            return_results(enrich_command(client, demisto.args()))
+
+    # Log exceptions and return errors
+    except Exception:
+        stacktrace = traceback.format_exc()
+        return_error(f'Error: {stacktrace}')
```
Comment (jbabazadeh) [inline Packs/SpurContextAPI/Integrations/SpurContextAPI/SpurContextAPI.py:158]:
```suggestion
        return_error(f'Error: {traceback.format_exc()}')
```

File: Packs/SpurContextAPI/.secrets-ignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+https://api.spur.us/
+192.168.1.1
```
Comment (jbabazadeh) [inline Packs/SpurContextAPI/.secrets-ignore:2]: why to use this in the tests? and not 1.1.1.1 instead?

**Review by jbabazadeh — CHANGES_REQUESTED** [review]
Hi @jjunqueira 
Thank you for your contribution!
Good work :)
Please see my comments

Please feel free to reach out to me with any questions - I'm available here or on slack :)

**Comment by ShirleyDenkberg** [comment]
I am on PTO through March 14. If this cannot wait, please contact @Richard
Bluestone ***@***.***> .

**Review by jbabazadeh — APPROVED** [review]

**Review by jbabazadeh — APPROVED** [review]

**Comment by jbabazadeh** [comment]
Hi @jjunqueira ,
I merged the internal PR, and your update is now in the master branch of content repo.
Your update should be available in the marketplace within the next 72 hours.
Again, thank you very much for your contribution!

**Pull request merged**